code/__DEFINES/role_preferences.dm

```diff
@@ -36,6 +36,7 @@
 #define ROLE_DEATHSQUAD			"Deathsquad"
 #define ROLE_LAVALAND			"Lavaland"
 #define ROLE_INTERNAL_AFFAIRS	"Internal Affairs Agent"
+#define ROLE_FUGITIVE           "Fugitive"
```

Comment:
Please. PLEASE make this inline with the rest

Reply:
retard it is